### PR TITLE
Add Playdate SDK 1.9.0

### PR DESCRIPTION
**Pull request by ngs**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
The `pkg` installs an `app` called `Playdate Simulator.app`, hence the name change.

**Comment by ngs** [comment]
@miccal This .pkg installer contains Lua + C Library, Docs and CLI tools. So I think it's better to name this cask `playdate-sdk` .

**Comment by miccal** [comment]
@ngs the naming follow our [strict token reference](https://docs.brew.sh/Cask-Cookbook#simplified-names-of-pkg-based-installers):

> If a pkg installs an App, then use that App name

File: Casks/playdate-simulator.rb [hunk 1]
```diff
@@ -1,18 +1,16 @@
-cask "playdate-sdk" do
+cask "playdate-simulator" do
   version "1.9.0"
   sha256 "c61e3ba45b65034f89690e5ad22b2ebd24e03df6d376b72e0fae62b003d104ff"
 
   url "https://download-keycdn.panic.com/playdate_sdk/PlaydateSDK-#{version}.pkg",
-      verified: "panic.com"
+      verified: "panic.com/playdate_sdk/"
```
Comment (neutric) [inline Casks/playdate-simulator.rb:6]: Shouldn't this be `verified: "download-keycdn.panic.com/playdate_sdk/"`?

Comment (miccal) [reply Casks/playdate-simulator.rb]: Yes it should, thanks.

**Comment by neutric** [comment]
> @miccal This .pkg installer also contains Lua + C Library, Docs and CLI tools.
> 
> So I think it's better to name this cask `playdate-sdk` .

This information is not lost as it is contained in `name` and `desc`. The `token` fulfills another [purpose](https://docs.brew.sh/Cask-Cookbook#purpose).

**Pull request merged**